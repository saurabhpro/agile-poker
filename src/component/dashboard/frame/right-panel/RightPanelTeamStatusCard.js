import React from 'react';
import { Card } from 'react-bootstrap';
import PropTypes from 'prop-types';

import Team from './team/Team';
import './RighSideCard.css';

const RightPanelTeamStatusCard = ({ style }) => {
  return (
    <Card className="rightCard teamCard" style={style}>
      <Card.Body>
        <Team className="title" />
      </Card.Body>
    </Card>
  );
};

RightPanelTeamStatusCard.propTypes = {
  style: PropTypes.object.isRequired,
};

export default RightPanelTeamStatusCard;
