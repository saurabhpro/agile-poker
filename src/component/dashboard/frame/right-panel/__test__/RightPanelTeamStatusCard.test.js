import React from 'react';
import { shallow } from 'enzyme';
import RightPanelTeamStatusCard from '../RightPanelTeamStatusCard';

let component;

describe('result from poker card panel', () => {
  beforeAll(() => {
    component = shallow(
      <RightPanelTeamStatusCard style={{ border: 'none' }} />,
    );
    //console.debug("This was prited",component.debug());
  });

  it('matches the rendered snapshot', () => {
    expect(component).toMatchSnapshot();
  });

  it('should render one <Team>', () => {
    expect(component.find('Team')).toHaveLength(1);
  });
});
