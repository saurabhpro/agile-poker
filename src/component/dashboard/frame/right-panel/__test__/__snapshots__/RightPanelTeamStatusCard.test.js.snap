// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`result from poker card panel matches the rendered snapshot 1`] = `
<Card
  body={false}
  className="rightCard teamCard"
  style={
    Object {
      "border": "none",
    }
  }
>
  <CardBody>
    <Team
      className="title"
    />
  </CardBody>
</Card>
`;
